src/ui/pages/HomePage/HomePage.ts

```diff
@@ -0,0 +1,84 @@
+import { Assets, Sprite } from 'pixi.js';
+import { BaseButton } from '../../components/button/BaseButton';
+import { BasePage } from '../BasePage';
+import { Bundles, GameAssets } from '../../../App.types';
+import { Events } from '../../../common/Events';
+import { HomePageStyles } from './HomePage.styles';
+import { PageIds } from '../Page.types';
+
+/**
+ * The main page of the app that has two CTAs;
+ * [PLAY] starts the game and [CREDITS] navigates to the credits page.
+ */
+export class HomePage extends BasePage {
+    private background: Sprite;
+    private creditsCTA: BaseButton;
+    private playCTA: BaseButton;
+
+    public constructor() {
+        console.log('HomePage().constructor()');
+        super(PageIds.HomePage);
+
+        this.init();
+    }
+
+    private init = async () => {
+        console.log('HomePage().init()');
+
+        await this.initPage();
+    }
+
+    private initPage = async () => {
+        console.log('HomePage().initPage()');
+
+        // set textures
+        const textures = await Assets.loadBundle(Bundles.GameAssets);
```

Comment:
It might be better to defile a Type for this. When I check the type on webstorm it says 'any'

Reply:
updated as suggested;

https://github.com/irgat/soloTest/pull/3/commits/e448abb350b2ae4c1bbb7cd45920a15713c2aee0